api/tests/e2e/orgs_test.go

```diff
@@ -211,6 +207,19 @@ var _ = Describe("Orgs", func() {
 			}).Should(Succeed())
 		})
 
+		When("the org contains a space", func() {
+			BeforeEach(func() {
+				createSpace(generateGUID("some-space"), orgGUID)
+			})
+
+			It("can still delete the org", func() {
+				Expect(resp).To(SatisfyAll(
+					HaveRestyStatusCode(http.StatusAccepted),
+					HaveRestyHeaderWithValue("Location", HaveSuffix("/v3/jobs/org.delete-"+orgGUID)),
```

Comment:
For the sake of proper E2E test, we should expect that eventually the URL specified by the `Location` header says that the job has completed successfully

Reply:
Good idea. We moved this test up but didn't change it in this PR, but we can take a stab at testing for job success.